addon/formatters/-base.js

```diff
@@ -65,7 +65,7 @@ const FormatBase = Ember.Object.extend({
 });
 
 FormatBase.reopenClass({
```

Comment:
Yeah I actually tried this approach first but then was worried I didn't really understand how broad of an impact a change here would have.